mongodbatlas/util/type_conversion.go

```diff
@@ -28,6 +28,15 @@ func Int64PtrToIntPtr(i64 *int64) *int {
 	return &i
 }
 
+func IntPtrToInt64Ptr(i *int) *int64 {
```

Comment:
this seems the inverse to admin.Int64PtrToIntPtr. , don't know if it makes sense to have it in the SDK

Reply:
this conversion only seems needed when using old sdk (that uses int type), so eventually will be removed.